### DISCUSSION TITLE
Filtering Rules

### DISCUSSION THREAD

**Discussion by Elliot-Construct** [Budibase/budibase#4919] [Ideas]
Hey Budies!

I would love to see a "contains" element to the filter logic processing.
and I would love to see an OR element to the filtering as well.

What do you folks think?

Cheers,
Elliot

**Comment by mjashanks** [comment]
Totally agree! 

"Contains" is an issue when using the internal database, as it is not supported by CouchDB - but there's no reason why we couldn't add it for SQL.

For the `OR` -  we would of course love to have this as well, but building the user interface for it is tricky. You can get around this at the moment by adding a `Formula` column (type: static). Then you can filter by the formula. I.e. put your `or` conditions inside the formula, and make the formula return a single value that you can filter on.
